dist/api_data.py

```diff
@@ -1373,6 +1373,9 @@ def __ge__(self, other):
         Display the contents of in-memory pages as they are verified,
         using the application's message handler, intended for debugging''',
         type='boolean'),
+    Config('hs_verify', 'false', r'''
+        Verify the history store''',
+        type='boolean'),
```

Comment:
Putting this comment in a random place, so please ignore the location, but why is `dist/clang-format` included in this pull request? It's a binary, right?